2021-03-18-if-else-exercises/main.js

```diff
@@ -0,0 +1,61 @@
+/* 
+    Exercise 1:
+    - complete the if/else statement so that the result is true if the input number is bigger equal than 70 and smaller equal than 99 
```

Comment:
```suggestion
    - Complete the if...else statement so that the result is true if the input number is bigger or equal than 70 and smaller or equal than 99 
```

Reply:
Should we use the MDN way of telling it, e.g. "greater than or equal to"?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal

Reply:
ah yes, that sounds good 👍